crates/bevy_ui/src/picking_backend.rs

```diff
@@ -19,10 +19,6 @@
 //!   camera.
 //! - To correctly sort picks, the order of `bevy_ui` is set to be the camera order plus 0.5.
 
-#![allow(clippy::type_complexity)]
-#![allow(clippy::too_many_arguments)]
-#![deny(missing_docs)]
```

Comment:
are we sure we want to remove `#![deny(missing_docs)]` if we have `#![expect(missing_docs, reason = "Not all docs are written yet, see #3492.")]` in lib?

Reply:
Honestly I didn't even mean to remove that line. I must've accidentally selected the `#![deny(missing_docs)]` line when removing the two allows.